tests/templates/logrotate.j2

```diff
@@ -1,13 +0,0 @@
-{{ nginx_log_path }}/*.log {
```

Comment:
is this file not needed anymore?

Reply:
It was an exact copy from the one in `templates`, and setting `nginx_logrotate_template_path: templates/logrotate.j2` makes the role to pick that one, so I think `tests/templates/logrotate.j2` is redundant and therefore not necessary anymore